pkg/capture/flow.go

```diff
@@ -84,30 +88,29 @@ func ParsePacket(ipLayer capture.IPLayer, pktType capture.PacketType, pktTotalLe
 
 		isIPv4, protocol = true, ipLayer[9]
 
-		// Parse IPv4 packet information
-		copy(epHash[0:4], ipLayer[12:16])
-		copy(epHash[16:20], ipLayer[16:20])
-
-		// only run the fragmentation checks on fragmented TCP/UDP packets. For
+		// Only run the fragmentation checks on fragmented TCP/UDP packets. For
 		// ESP, we don't have any transport layer information so there's no
 		// need to distinguish between ESP fragments or other ESP traffic
 		//
 		// Note: an ESP fragment will carry fragmentation information like any
 		// other IP packet. The fragment offset will of be MTU - 20 bytes (IP layer).
 		if protocol != capturetypes.ESP {
 
-			// check for IP fragmentation
+			// Check for IP fragmentation
 			fragOffset := (uint16(0x1f&ipLayer[6]) << 8) | uint16(ipLayer[7])
 
-			// return decoding error if the packet carries anything other than the
-			// first fragment, i.e. if the packet lacks a transport layer header
+			// Skip packet if it carries anything other than the first fragment,
+			// i.e. if the packet lacks a transport layer header
 			if fragOffset != 0 {
-				fragBits := (0xe0 & ipLayer[6]) >> 5
-				err = fmt.Errorf("fragmented IP packet: offset: %d flags: %d", fragOffset, fragBits)
+				err = ErrPacketFragmentIgnore
```

Comment:
Still need to signify as an error so we can skip ahead in `(f *FlowLog) Add()` for such cases.